cfpb/cfgov-refresh
# Add regulation metadata to section pages

Add regulation metadata, like the regulation's effective date and search link, to section pages in the regulation.

## Additions

- Search link to regulation section pages
- Effective date to regulation section pages

## Testing

1. Fire up Regulations 3000 as you usually do
2. Go to any section, appendix, or supplement page
3. Verify that the search link shows up above the page's `h1` and points to `#` (for now; see the todo below)
4. Verify that the regulation's effective date shows up under the page's `h1`

## Screenshots

![regs-metadata](https://user-images.githubusercontent.com/1862695/42229960-bf3baba4-7eb5-11e8-9f37-db9ece1ec60e.png)

## Todos

- The search link doesn't point to anywhere right now. We're still figuring out the routes for the search page; once we settle on those URLs, we'll need to update the `href` in the search link to point to the right place.

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
